### PR TITLE
Review

### PR DESCRIPTION
**Pull request by oovillagran**
Create a new empty repo.
 Copy your code from the "To Do list" project and commit it as one big chunk of code in a feature branch.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    }
+  });
+});
+
+const clearButton = document.getElementById('clear-button');
+clearButton.addEventListener('click', () => {
+  taskManager.taskList = taskManager.taskList.filter((task) => !task.completed);
+  taskManager.saveTasks();
+  window.location.reload();
+});
```
Comment (federicaulzurrun) [inline index.js]: - [ ] The code lacks comments that explain what each method or function does, which makes it difficult to understand for other developers.

- [ ] Variable names: Some variable and method names are not very descriptive and could be improved to make the code more readable.

**Review by federicaulzurrun — COMMENTED** [review]
tufoin

Hi Oscar!! First of all congratulation on your project!

- [X] Your JavaScript code follows the DRY, KISS, and YANGI rules
- [X]  You followed all the HTML and CSS good practices

I suggest you give it a look again to [JavaScript Best Practices](urhttps://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.mdl)

Check the comments below for suggestions so you can follow the JavaScript bests practices!
@ me in the comments, if you need any help understanding my suggestions!

File: index.js (hunk 1 above)
Comment (oovillagran) [reply index.js]: Thank you @federicaulzurrun, I appreciate your comments.

**Review by federicaulzurrun — APPROVED** [review]
it's time to merge!!!

**Pull request merged**